Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -252,6 +252,9 @@ ActsExamples::ProcessCode ActsExamples::SeedingAlgorithm::execute(
   seeds.clear();
   static thread_local decltype(m_seedFinder)::SeedingState state;
 
+  state.spacePointData.clear();
```

Comment:
I suppose we can implement something like this. Possibly accepting a flag that would specify if we are running on pixel or strips. I would however prefer to implement this in #1966 instead of here

Reply:
Definitely ok.